RELEASES.md

```diff
@@ -385,6 +385,8 @@ The deprecated features are shown in the following table:
 | `cri-containerd-*.tar.gz` release bundles                                        | containerd v1.6     | containerd v2.0            | Use `containerd-*.tar.gz` bundles        |
 | Pulling Schema 1 images (`application/vnd.docker.distribution.manifest.v1+json`) | containerd v1.7     | containerd v2.0            | Use Schema 2 or OCI images               |
 | CRI `v1alpha2`                                                                   | containerd v1.7     | containerd v2.0 ✅         | Use CRI `v1`                             |
+| Legacy CRI implementation                                                        | containerd v2.0     | containerd v2.1            | Enabled by default in 2.0 (use `DISABLE_CRI_SANDBOXES=1` to fallback) |
```

Comment:
"Legacy CRI implementation with pause containers" or "Legacy non-sandbox CRI implementation" might be more descriptive